ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -168,7 +168,8 @@ private async Task<HospitalDetails> ExtractHospitalDetailsAsync(dynamic rawNotif
             var details = new HospitalDetails();
             details.HospitalId = rawNotification.NtbsHospitalId;
             details.CaseManagerUsername = rawNotification.CaseManager;
-            details.Consultant = rawNotification.Consultant;
+            var consultant = RemoveCharactersNotIn(ValidationRegexes.CharacterValidationWithNumbersForwardSlashExtended, rawNotification.Consultant);
```

Comment:
When I was looking at doing this Jam mentioned that it might be nice to make this an extension method rather than a private method.

I think if we were going to do that, we would want to do fiddle with the format of the ValidationRegexes (as currently there's some specifc logic in `RemoveCharactersNotIn` to transform the regex). Which I'm not entirely convinced is worth it. 

Thought I would bring it up though.